### PR TITLE
Port supported languages API from Neon speech/audio modules

### PR DESCRIPTION
**Pull request by NeonDaniel**

### CODE REVIEW

**Comment by NeonDaniel** [comment]
https://github.com/NeonGeckoCom/neon_speech/pull/95
https://github.com/NeonGeckoCom/neon_audio/pull/86

File: mycroft/audio/service.py [hunk 1]
```diff
@@ -218,6 +218,16 @@ def handle_queue_audio(self, message):
...
 
+    def handle_get_languages_tts(self, message):
+        """
+        Handle a request for supported TTS languages
+        :param message: neon.get_languages_tts request
+        """
+        tts_langs = self.tts.available_languages or \
+            [self.config.get('language', {}).get('user') or 'en-us']
+        LOG.info(f"Got tts_langs: {tts_langs}")
+        self.bus.emit(message.response({'tts_langs': tts_langs}))
```
Comment (JarbasAl) [inline mycroft/audio/service.py:229]: can we unify this data field as "langs" and change only the message type between services?

also update the log to DEBUG

File: mycroft/audio/service.py [hunk 2]
```diff
@@ -238,3 +248,4 @@ def init_messagebus(self):
         self.bus.on('mycroft.audio.speech.stop', self.handle_stop)
         self.bus.on('mycroft.audio.queue', self.handle_queue_audio)
         self.bus.on('speak', self.handle_speak)
+        self.bus.on('neon.get_languages_tts', self.handle_get_languages_tts)
```
Comment (JarbasAl) [inline mycroft/audio/service.py:251]: can we change this message? let's also try to standardize the message a little bit `ovos.languages.{service}.get`?

File: mycroft/listener/service.py [hunk 3]
```diff
@@ -191,6 +191,16 @@ def handle_open(self):
...
 
+    def handle_get_languages_stt(self, message):
+        """
+        Handle a request for supported STT languages
+        :param message: neon.get_languages_stt request
+        """
+        stt_langs = self.loop.stt.available_languages or \
+            [self.config.get('language', {}).get('user') or 'en-us']
+        LOG.info(f"Got stt_langs: {stt_langs}")
+        self.bus.emit(message.response({'stt_langs': stt_langs}))
```
Comment (JarbasAl) [inline mycroft/listener/service.py:202]: same as above comment for TTS

File: mycroft/listener/service.py [hunk 4]
```diff
@@ -221,6 +231,7 @@ def connect_bus_events(self):
                     self.handle_audio_start)
         self.bus.on('recognizer_loop:audio_output_end', self.handle_audio_end)
         self.bus.on('mycroft.stop', self.handle_stop)
+        self.bus.on("neon.get_languages_stt", self.handle_get_languages_stt)
```
Comment (JarbasAl) [inline mycroft/listener/service.py:234]: same as above comment for TTS message

File: mycroft/audio/service.py (hunk 1 above)
Comment (NeonDaniel) [reply mycroft/audio/service.py:229]: makes sense; that will simplify parsing on the other end as well

**Pull request merged**